src/Illuminate/Foundation/Testing/Concerns/InteractsWithDatabase.php

```diff
@@ -189,11 +189,11 @@ public function castAsJson($value)
      * @param  string|null  $connection
      * @return \Illuminate\Database\Connection
```

Comment:
@thibodelanghe: `@param  \Illuminate\Database\Eloquent\Model|string  $table`

Reply:
Yes it does, thanks! Opened new PR https://github.com/laravel/framework/pull/40496

Reply:
@thibodelanghe: other docblocks permit `$table` to be an instance of `\Illuminate\Database\Eloquent\Model`, should your PR include this?